client/components/domains/use-your-domain-step/index.jsx

```diff
@@ -1,469 +0,0 @@
-		}
-	};
-
-	getMappingPriceText = () => {
-		const {
-			cart,
-			currencyCode,
-			domainsWithPlansOnly,
-			primaryWithPlansOnly,
-			productsList,
-			selectedSite,
-			translate,
-		} = this.props;
-		const { searchQuery } = this.state;
-
-		let mappingProductPrice;
-
-		const price = get( productsList, [ 'domain_map', 'cost' ], null );
-		if ( price ) {
-			mappingProductPrice = formatCurrency( price, currencyCode );
-			mappingProductPrice = translate(
-				'%(cost)s per year plus registration costs at your current provider',
-				{ args: { cost: mappingProductPrice } }
-			);
-		}
-
-		if (
-			isDomainMappingFree( selectedSite ) ||
-			isNextDomainFree( cart ) ||
-			isDomainBundledWithPlan( cart, searchQuery )
-		) {
-			mappingProductPrice = translate(
-				'Free with your plan, but registration costs at your current provider still apply'
-			);
-		} else if ( domainsWithPlansOnly || primaryWithPlansOnly ) {
-			mappingProductPrice = translate(
-				'Included in annual paid plans, but registration costs at your current provider still apply'
-			);
-		}
-
-		return mappingProductPrice;
-	};
-
-	renderIllustration = ( image ) => {
-		return (
-			<div className="use-your-domain-step__option-illustration">
-				<img src={ image } alt="" />
-			</div>
-		);
-	};
-
-	renderOptionTitle = ( optionTitle ) => {
-		return <h3 className="use-your-domain-step__option-title">{ optionTitle }</h3>;
-	};
-
-	renderOptionReasons = ( optionReasons ) => {
-		return (
-			<div className="use-your-domain-step__option-reasons">
-				{ optionReasons.map( ( phrase, index ) => {
-					if ( isEmpty( phrase ) ) {
-						return;
-					}
-
-					return (
-						<div className="use-your-domain-step__option-reason" key={ index }>
-							<Gridicon icon="checkmark" size={ 18 } />
-							{ phrase }
-						</div>
-					);
-				} ) }
-			</div>
-		);
-	};
-
-	renderOptionContent = ( content ) => {
-		const { image, title, reasons, onClick, buttonText, isPrimary, learnMore } = content;
-		return (
-			<Card className="use-your-domain-step__option" compact>
-				<div className="use-your-domain-step__option-inner-wrap">
-					<div className="use-your-domain-step__option-content">
-						{ this.renderIllustration( image ) }
-						{ this.renderOptionTitle( title ) }
-						{ this.renderOptionReasons( reasons ) }
-					</div>
-					<div className="use-your-domain-step__option-action">
-						{ this.renderOptionButton( { onClick, buttonText, isPrimary } ) }
-						<div className="use-your-domain-step__learn-more">{ learnMore }</div>
-					</div>
-				</div>
-			</Card>
-		);
-	};
-
-	renderOptionButton = ( buttonOptions ) => {
-		const { buttonText, onClick, isPrimary } = buttonOptions;
-		const { submittingAvailability, submittingWhois } = this.state;
-		const submitting = submittingAvailability || submittingWhois;
-		return (
-			<Button
-				className="use-your-domain-step__option-button"
-				primary={ isPrimary }
-				onClick={ onClick }
-				busy={ submitting }
```

Comment:
I removed the `busy` prop since, in the current implementation, it was always computing as `false`